code/game/objects/items/holy_weapons.dm

```diff
@@ -468,6 +468,62 @@
 		qdel(S)
 	return ..()
 
+/obj/item/nullrod/scythe/talking/dummy
+	name = "possessed dummy"
+	desc = "It's a dummy, dummy. It glares at you with cold, unblinking eyes."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "assistant"
+	item_state = "doll"
+	attack_verb = list("bit", "slapped", "mocked")
+	hitsound = "swing_hit"
+	possessed = FALSE
+	force = 5
+	throwforce = 3
+	w_class = WEIGHT_CLASS_TINY
+	sharpness = IS_BLUNT
+	armour_penetration = 0
+	slot_flags = 0
+	var/doll_name = "Dummy"
+
+/obj/item/nullrod/scythe/talking/dummy/attack_self(mob/living/user)
+	if(possessed)
+		return
+
+	to_chat(user, "<span class='notice'>You attempt to wake the spirit of the dummy...</span>")
+
+	possessed = TRUE
+
+	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to play as the spirit of [user.real_name]'s dummy?", ROLE_PAI, null, FALSE, 100, POLL_IGNORE_POSSESSED_BLADE)
+
+	if(LAZYLEN(candidates))
```

Comment:
im guessing this is all copypasted from the possessed blade which is bad

Reply:
most of it is, with some edits, yes